ssm_session_policy.tf

```diff
@@ -0,0 +1,73 @@
+# ------------------------------------------------------------------------------
+# Create the IAM policy that allows creation of SSM SessionManager
+# sessions to any EC2 instance in this account.
+# ------------------------------------------------------------------------------
+
+data "aws_iam_policy_document" "ssm_session_doc" {
+  # Allow the user to start a session
+  statement {
+    actions = [
+      "ssm:SendCommand",
+      "ssm:StartSession",
+    ]
+    resources = [
+      "arn:aws:ec2:${var.aws_region}:${local.this_account_id}:instance/*",
+      "arn:aws:ssm:${var.aws_region}::document/AWS-StartInteractiveCommand",
+      "arn:aws:ssm:${var.aws_region}::document/AWS-StartNonInteractiveCommand",
+      "arn:aws:ssm:${var.aws_region}::document/AWS-StartPortForwardingSession",
+      "arn:aws:ssm:${var.aws_region}::document/AWS-StartPortForwardingSessionToSocket",
+      "arn:aws:ssm:${var.aws_region}:${local.this_account_id}:document/SSM-SessionManagerRunShell",
```

Comment:
It might be nice to document in the README which sessions can be started after running this module.  Sure, they could just look in the code, so take it or leave it on this suggestion.

Reply:
Please see commit 18905c7.

Reply:
I like it - thanks! 👍